.circleci/config.yml

```diff
@@ -303,6 +303,39 @@ references:
         export BACKUP_SUFFIX=${BACKUP_ENV:?}_db_backup_$(date +%s)
         cd /infrastructure && make mnesia_backup
 
+  test_steps: &test_steps
+    - checkout
+    - *restore_rebar_cache
+    - *restore_build_cache
+    - run:
+        name: Test
+        command: |
+          epmd -daemon
+          make ${MAKE_TARGET:-ct} CT_TEST_FLAGS="--suite=$(.circleci/split_suites.sh)"
```

Comment:
I am a bit concerned of the default ending up - e.g. for typo - running same target for all configs. What about `${MAKE_TARGET:?}`?